platforms/core-execution/persistent-cache/build.gradle.kts

```diff
@@ -28,11 +28,10 @@ dependencies {
     api(project(":messaging"))
     api(project(":native"))
     api(project(":files"))
-    api(project(":logging"))
```

Comment:
💭 This is also important: `:persistent-cache` has no business with logging, and now that's actually the case.